src/LsifSymbol.ts

```diff
@@ -0,0 +1,37 @@
+import { Descriptor } from './Descriptor'
+
+export class LsifSymbol {
+  private constructor(public readonly value: string) {}
+  public isEmpty(): boolean {
+    return this.value === ''
+  }
+  public isLocal(): boolean {
+    return this.value.startsWith('local ')
+  }
+  public isEmptyOrLocal(): boolean {
+    return this.isEmpty() || this.isLocal()
+  }
+
+  public static local(counter: number): LsifSymbol {
+    return new LsifSymbol(`local ${counter}`)
+  }
+  public static empty(): LsifSymbol {
+    return new LsifSymbol('')
+  }
+  public static sourceFile(
+    package_: LsifSymbol,
+    relativePath: string[]
+  ): LsifSymbol {
+    let symbol = package_
+    for (const part of relativePath) {
+      symbol = LsifSymbol.global(symbol, Descriptor.package(part))
+    }
+    return symbol
+  }
```

Comment:
It looks like this method is unused... Could you turn on warnings (and make it an error in CI) for unused variables?